tools/cldr-apps/js/src/esm/cldrProgress.mjs

```diff
@@ -366,7 +366,7 @@ function fetchLocaleData(unlessLoaded) {
     return;
   }
   const locale = cldrStatus.getCurrentLocale();
-  if (!locale) {
+  if (!locale || locale === "USER") {
```

Comment:
possible future enhancement, reject if getLocaleInfo returns null

Reply:
basically centralize this code… yeah